### ISSUE TITLE
r30gb does not work on Yi Dome 1080p BFUS serial

### ISSUE THREAD

**Issue by t00** [roleoroleo/yi-hack-Allwinner-v2#122]
I am trying to run the hack but the camera won't start. Clearly blue WiFi led is settling on the on state after blinking a few times but the camera does not do calibration.
I received camera today with intent to run custom fw, which I can see is beta. Is there anything worth trying to make it work?
Does it make a difference if I copy r30gb release contents to 32GB SD card formatted using a camera but on Linux?
Obviously after removing SD card camera works fine. Current firmware is 4.6.0.0A_201908271549.

**Comment by t00** [comment]
The camera creates log files when WiFi connection is established and Reset is pressed  (names them tar.gz but it is plain tar - I recompressed these after removing my wifi pass):
[start_reset.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6253429/start_reset.tar.gz)
[wifi_connected.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/6253430/wifi_connected.tar.gz)

@roleoroleo I have a suspicion the software version number 4.6.0.0A is more likely to be MStar based on other v4 threads, yet the serial number of the latest model of Dome 1080p (YHS_2016) starts with BFUS pointing to r30gb.

I also tried backup.tar.gz and Factory dir on FAT32 SD did not change name nor backup was created after plugging in power.
Anything else worth trying?
Log files contain list of files/modules.
